types/next-auth/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "target": "es6",
```

Comment:
I'm unsure about the side-effects of this change 🤔 , in theory, we should match to whatever the library is compiling to. They use [**Babel**](https://github.com/nextauthjs/next-auth/blob/main/config/babel.config.json) but I don't think transpiling to ES6 directly? ( cc. @balazsorban44 could help us 💡 )

Reply:
I'm not sure either, i just modified this and the typescript version [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52116/files/44e8255ef21db58b4165c7e310cb3ce24e57a79c#diff-9fda210ef628d559546e578a73049a164e92d7c27280ce4301e4076745300db1L9-R9), because they were giving me errors when testing the library, and I thought that it needed to be corrected somehow

Reply:
Isn't this only for the type compilation? We don't use that config anywhere in the main repo, so I guess it doesn't matter what is its value as long as it works. 🤷